### PR TITLE
Translating 8 term to portuguese

### PR DESCRIPTION
**Pull request by marcosvital**
## Author: 

- @marcosvital

## Language: 

- Portuguese

## Terms defined:

- absolute_error
- absolute_row_number
- aggregation
- argument
- arithmetic_mean
- mean
- relative_error
- relative_row_number

## Editor

@beatrizmilz

### CODE REVIEW

File: glossary.yml [hunk 1]
```diff
@@ -181,6 +191,11 @@
     def: >
       Om 'n groep waardes in een te kombineer bv. om 'n versameling getalle op
       te som of om 'n versameling alfabetiese stringe aaneen te skakel.
+  pt:
+    term: "agregação"
+    def: >
+      Combinar vários valores em um, como a soma de uma série de números ou
+      a combinação de mais de um conjunto de caracteres.
```
Comment (beatrizmilz) [inline glossary.yml:198]:
```suggestion
      concatenando um conjunto de caracteres.
```

File: glossary.yml [hunk 2]
```diff
@@ -291,12 +306,21 @@
     def: >
       Un valor pasado a una función. Algunos autores usan el término como
       sinónimo de [parámetro](#parameter) y algunos no; Todo es muy confuso.
+  pt:
+    term: "argumento"
+    def: >
+      Um valor passado para uma função. Alguns autores usam este termo como sinônimo
```
Comment (beatrizmilz) [inline glossary.yml:312]:
```suggestion
      Um valor passado para uma função. Algumas pessoas autoras usam este termo como sinônimo
```

File: glossary.yml [hunk 3]
```diff
@@ -291,12 +306,21 @@
...
+      de parâmetro(#parameter) e outro não fazem isso; é tudo bastante confuso.
```
Comment (beatrizmilz) [inline glossary.yml:313]:
```suggestion
      de parâmetro(#parameter) e outras não fazem isso; é tudo bastante confuso.
```

**Review by beatrizmilz — COMMENTED** [review]
Fiz pequenas sugestões, o que acha?
Em uma das sugestões, mudei o gênero da palavra autores, que é uma sugestão do guia de contribuição em Espanhol (ainda não temos um guia deste tipo em Português, porém acho muito interessante os direcionamentos). Em vez de 'autores', mudei para 'pessoas autoras'.

**Comment by marcosvital** [comment]
Perfeitas as sugestões, Beatriz, obrigado! Ficarei atento para o uso do gênero de palavras como autores - achei bacana o formato proposto.

**Comment by beatrizmilz** [comment]
Thank you @marcosvital ! @fmichonneau All the translation is great, but GitHub shows that there is a conflict in several lines. I'm not sure what is causing that, do you know?

**Comment by zkamvar** [comment]
> All the translation is great, but GitHub shows that there is a conflict in several lines. I'm not sure what is causing that, do you know why?

@beatrizmilz This is because a french translation of these terms was added and merged before this PR was accepted and Git doesn't know which one is correct. This is an interesting merge because you want to blend the two together. 

If you have the permissions, there should be a button below that says "Resolve conflicts". If you click on that button, GitHub will open an editor that will highlight the conflicts, which you can fix like this:

conflict-resolution

**Comment by fmichonneau** [comment]
@beatrizmilz Zhian's screencast demonstrates the steps to take so give it a try! (thanks for doing that Zhian!)
After doing this, click on the "commit changes" at the top of the editor, make sure the checks still pass, and then you can approve and merge the PR.
Let us know if you need help. Thank you!

**Comment by beatrizmilz** [comment]
@zkamvar  Thanks so much for the explanation and for creating this screencast. It helped a lot!

I was just a bit insecure because I fixed the conflits and this message apeared:
image

I was afraid to commit to `carpentries:master` before I could see if the check is OK, but I looks like it in was `marcosvital:master`.
The checks are ok now, I'll try to merge it :)

**Review by beatrizmilz — APPROVED** [review]

**Comment by beatrizmilz** [comment]
@zkamvar @fmichonneau Seems like is merged! Can any of you check if I did everything right? Thank you for all the help and instructions!

and thanks @marcosvital for the translations!

**Comment by fmichonneau** [comment]
everything looks good! Thank you all.

**Comment by zkamvar** [comment]
> I was afraid to commit to `carpentries:master` before I could see if the check is OK, but I looks like it in was `marcosvital:master`.
> The checks are ok now, I'll try to merge it :)

This is a common pattern that reflects on the historically poor documentation for [GitHub Flow](https://guides.github.com/introduction/flow/) (which only describes the branching aspect, but doesn't delve into the forking aspect. The [Happy Git With R tutorial](https://happygitwithr.com/fork-and-clone.html#fork-and-clone) by Dr. Jenny Bryan does a REALLY good job of introducing this clearly, but it's definitely focused on R users. 

If you try to edit the file directly on github, you will often find that it will automatically create a fork and new branch for you, which is helpful, but kind of obfuscates the process.

This is all to say, thank you @marcosvital for you contribution and thank you @beatrizmilz for handling the merge conflict (which threw more than one curvball!)

**Comment by elletjies** [comment]
@all-contributors please add @marcosvital for translation

**Pull request merged**